File: course-p-oop/POOUber/PHP/uberVan.php
<?php

require_once('car.php');

class UberVan extends Car{
    public $typeCarAccepted;
    public $seatsMaterial;

    public function _construct($license, $driver, $typeCarAccepted, $seatsMaterial)
    {
        parent::__construct($license, $driver);
        $this->seatsMaterial = $seatsMaterial;
        $this->seatsMaterial = $seatsMaterial;
    }

    public function setPassenger($passenger){
        if ($passenger == 6){
            $this->passenger = $passenger;
            echo "6 pasajeros asignados.";
        }
        else{
            echo "Debes ingresar 6 pasajeros.";
        }
    }
}

?>

File: course-p-oop/POOUber/Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car{
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    private Integer passenger;
  

    public UberVan(String license, Account driver) {
        super(license, driver);
    }
    
    @Override
    public void SetPassenger(Integer passenger){
        if (passenger == 6){
            this.passenger = passenger;
            System.out.println("6 pasajeros asignados");
        }
        else{
            System.out.println("Solo se aceptan 6 pasajeros.");
        }
    }
}
